### PR TITLE
purge app except app host

### PR DESCRIPTION
**Pull request by jamesmontemagno**
Purge out anything ".NET Aspire app" and replace with .NET Aspire project.

Keep all "app host" or "AppHost" as this is what it is called... this project is hosting your apps.

---

#### Internal previews

<details><summary><strong>Toggle expand/collapse</strong></summary><br/>

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/caching/caching-components-deployment.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/caching/caching-components-deployment.md) | [Tutorial: Deploy a .NET Aspire project with a Redis Cache to Azure](https://review.learn.microsoft.com/en-us/dotnet/aspire/caching/caching-components-deployment?branch=pr-en-us-1105) |
| [docs/database/seed-database-data.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/database/seed-database-data.md) | [Seed data in a database using .NET Aspire](https://review.learn.microsoft.com/en-us/dotnet/aspire/database/seed-database-data?branch=pr-en-us-1105) |
| [docs/database/sql-server-component-deployment.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/database/sql-server-component-deployment.md) | [Tutorial: Deploy a .NET Aspire project with a SQL Server Database to Azure](https://review.learn.microsoft.com/en-us/dotnet/aspire/database/sql-server-component-deployment?branch=pr-en-us-1105) |
| [docs/deployment/azure/aca-deployment-azd-in-depth.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/deployment/azure/aca-deployment-azd-in-depth.md) | [[Windows](#tab/windows)](https://review.learn.microsoft.com/en-us/dotnet/aspire/deployment/azure/aca-deployment-azd-in-depth?branch=pr-en-us-1105) |
| [docs/deployment/azure/aca-deployment-github-actions.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/deployment/azure/aca-deployment-github-actions.md) | [[Windows](#tab/windows)](https://review.learn.microsoft.com/en-us/dotnet/aspire/deployment/azure/aca-deployment-github-actions?branch=pr-en-us-1105) |
| [docs/deployment/azure/aca-deployment-visual-studio.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/deployment/azure/aca-deployment-visual-studio.md) | [docs/deployment/azure/aca-deployment-visual-studio](https://review.learn.microsoft.com/en-us/dotnet/aspire/deployment/azure/aca-deployment-visual-studio?branch=pr-en-us-1105) |
| [docs/deployment/azure/aca-deployment.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/deployment/azure/aca-deployment.md) | [Deploy a .NET Aspire project to Azure Container Apps](https://review.learn.microsoft.com/en-us/dotnet/aspire/deployment/azure/aca-deployment?branch=pr-en-us-1105) |
| [docs/deployment/azure/application-insights.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/deployment/azure/application-insights.md) | [Use .NET Aspire with Application Insights](https://review.learn.microsoft.com/en-us/dotnet/aspire/deployment/azure/application-insights?branch=pr-en-us-1105) |
| [docs/deployment/azure/custom-bicep-templates.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/deployment/azure/custom-bicep-templates.md) | [Use custom Bicep templates](https://review.learn.microsoft.com/en-us/dotnet/aspire/deployment/azure/custom-bicep-templates?branch=pr-en-us-1105) |
| [docs/deployment/manifest-format.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/deployment/manifest-format.md) | [.NET Aspire manifest format for deployment tool builders](https://review.learn.microsoft.com/en-us/dotnet/aspire/deployment/manifest-format?branch=pr-en-us-1105) |
| [docs/deployment/overview.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/deployment/overview.md) | [docs/deployment/overview](https://review.learn.microsoft.com/en-us/dotnet/aspire/deployment/overview?branch=pr-en-us-1105) |
| [docs/extensibility/custom-resources.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/extensibility/custom-resources.md) | [Create custom resource types for .NET Aspire](https://review.learn.microsoft.com/en-us/dotnet/aspire/extensibility/custom-resources?branch=pr-en-us-1105) |
| [docs/frameworks/dapr.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/frameworks/dapr.md) | [Use Dapr with .NET Aspire](https://review.learn.microsoft.com/en-us/dotnet/aspire/frameworks/dapr?branch=pr-en-us-1105) |
| [docs/fundamentals/app-host-overview.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/fundamentals/app-host-overview.md) | [.NET Aspire orchestration overview](https://review.learn.microsoft.com/en-us/dotnet/aspire/fundamentals/app-host-overview?branch=pr-en-us-1105) |
| [docs/fundamentals/components-overview.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/fundamentals/components-overview.md) | [docs/fundamentals/components-overview](https://review.learn.microsoft.com/en-us/dotnet/aspire/fundamentals/components-overview?branch=pr-en-us-1105) |
| [docs/fundamentals/dashboard/explore.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/fundamentals/dashboard/explore.md) | [Explore the .NET Aspire dashboard](https://review.learn.microsoft.com/en-us/dotnet/aspire/fundamentals/dashboard/explore?branch=pr-en-us-1105) |
| [docs/fundamentals/persist-data-volumes.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/fundamentals/persist-data-volumes.md) | [Persist data with .NET Aspire using volume mounts](https://review.learn.microsoft.com/en-us/dotnet/aspire/fundamentals/persist-data-volumes?branch=pr-en-us-1105) |
| [docs/fundamentals/setup-tooling.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/fundamentals/setup-tooling.md) | [.NET Aspire setup and tooling](https://review.learn.microsoft.com/en-us/dotnet/aspire/fundamentals/setup-tooling?branch=pr-en-us-1105) |
| [docs/fundamentals/telemetry.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/fundamentals/telemetry.md) | [docs/fundamentals/telemetry](https://review.learn.microsoft.com/en-us/dotnet/aspire/fundamentals/telemetry?branch=pr-en-us-1105) |
| [docs/fundamentals/testing.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/fundamentals/testing.md) | [Testing .NET Aspire projects](https://review.learn.microsoft.com/en-us/dotnet/aspire/fundamentals/testing?branch=pr-en-us-1105) |
| [docs/get-started/aspire-overview.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/get-started/aspire-overview.md) | [.NET Aspire overview](https://review.learn.microsoft.com/en-us/dotnet/aspire/get-started/aspire-overview?branch=pr-en-us-1105) |
| [docs/get-started/build-aspire-apps-with-nodejs.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/get-started/build-aspire-apps-with-nodejs.md) | [Add Node.js apps to a .NET Aspire project](https://review.learn.microsoft.com/en-us/dotnet/aspire/get-started/build-aspire-apps-with-nodejs?branch=pr-en-us-1105) |
| [docs/messaging/messaging-components.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/messaging/messaging-components.md) | [Tutorial: Use .NET Aspire messaging components in ASP.NET Core](https://review.learn.microsoft.com/en-us/dotnet/aspire/messaging/messaging-components?branch=pr-en-us-1105) |
| [docs/storage/azure-storage-components.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/storage/azure-storage-components.md) | [docs/storage/azure-storage-components](https://review.learn.microsoft.com/en-us/dotnet/aspire/storage/azure-storage-components?branch=pr-en-us-1105) |
| [docs/whats-new/dotnet-docs-aspire-mod0.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/whats-new/dotnet-docs-aspire-mod0.md) | [.NET Aspire docs: What's new for March 2024](https://review.learn.microsoft.com/en-us/dotnet/aspire/whats-new/dotnet-docs-aspire-mod0?branch=pr-en-us-1105) |
| [docs/whats-new/dotnet-docs-aspire-mod1.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/whats-new/dotnet-docs-aspire-mod1.md) | [".NET Aspire docs: What's new for April 2024"](https://review.learn.microsoft.com/en-us/dotnet/aspire/whats-new/dotnet-docs-aspire-mod1?branch=pr-en-us-1105) |
| [docs/whats-new/dotnet-docs-aspire-mod2.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/whats-new/dotnet-docs-aspire-mod2.md) | [.NET Aspire docs: What's new for May 2024](https://review.learn.microsoft.com/en-us/dotnet/aspire/whats-new/dotnet-docs-aspire-mod2?branch=pr-en-us-1105) |
| [docs/whats-new/preview-3.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/whats-new/preview-3.md) | [docs/whats-new/preview-3](https://review.learn.microsoft.com/en-us/dotnet/aspire/whats-new/preview-3?branch=pr-en-us-1105) |
| [docs/whats-new/preview-4.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/whats-new/preview-4.md) | [.NET Aspire preview 4](https://review.learn.microsoft.com/en-us/dotnet/aspire/whats-new/preview-4?branch=pr-en-us-1105) |
| [docs/whats-new/preview-5.md](https://github.com/dotnet/docs-aspire/blob/c55da17f34545fe72546d92adefc095f82f5257c/docs/whats-new/preview-5.md) | [.NET Aspire preview 5](https://review.learn.microsoft.com/en-us/dotnet/aspire/whats-new/preview-5?branch=pr-en-us-1105) |

</details>

> [!NOTE]
> This table shows preview links for the 30 files with the most changes. For preview links for other files in this PR, select <strong>OpenPublishing.Build Details</strong> within [checks](https://github.com/dotnet/docs-aspire/pull/1105/checks).

### CODE REVIEW

File: docs/deployment/azure/aca-deployment-azd-in-depth.md [hunk 1]
```diff
@@ -54,23 +54,23 @@ The `azd init` workflow provides customized supported for .NET Aspire projects.
 1. Finally, once the container image is in ACR, `azd` updates the resource using ARM to start using the new version of the container image.
 
 > [!NOTE]
-> `azd` also enables you to output the generated Bicep to an `infra` folder in your project, which you can read more about in the [Generating Bicep from .NET Aspire app model](/dotnet/aspire/deployment/azure/aca-deployment-azd-in-depth?branch=main#generate-bicep-from-net-aspire-app-model) section.
+> `azd` also enables you to output the generated Bicep to an `infra` folder in your project, which you can read more about in the [Generating Bicep from .NET Aspire project model](/dotnet/aspire/deployment/azure/aca-deployment-azd-in-depth?branch=main#generate-bicep-from-net-aspire-app-model) section.
```
Comment (IEvangelist) [inline docs/deployment/azure/aca-deployment-azd-in-depth.md:57]: The "app model" needs to remain as is too, it's defined in the app host article.

```suggestion
> `azd` also enables you to output the generated Bicep to an `infra` folder in your project, which you can read more about in the [Generating Bicep from .NET Aspire app model](/dotnet/aspire/deployment/azure/aca-deployment-azd-in-depth?branch=main#generate-bicep-from-net-aspire-app-model) section.
```

File: docs/fundamentals/setup-tooling.md [hunk 2]
```diff
@@ -169,7 +169,7 @@ These templates matched your input: 'aspire'
 Template Name                     Short Name              Language  Tags
 --------------------------------  ----------------------  --------  -------------------------------------------------------
 .NET Aspire App Host              aspire-apphost          [C#]      Common/.NET Aspire/Cloud
-.NET Aspire Application           aspire                  [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service
+.NET Aspire project           aspire                  [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service
```
Comment (IEvangelist) [inline docs/fundamentals/setup-tooling.md:172]: This is the template name. Needs to remain as is...

```suggestion
.NET Aspire Application           aspire                  [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service
```

**Review by IEvangelist — APPROVED** [review]
LGTM, except a few places need to be reverted.

File: docs/fundamentals/setup-tooling.md (hunk 2 above)
Comment (jamesmontemagno) [reply docs/fundamentals/setup-tooling.md:172]: nice one... we should rename this...... I will send out an email

creates an app host and service defaults... doesn't seem super necessary ..

**Pull request merged**